cirq/experiments/random_quantum_circuit_generation.py

```diff
@@ -289,16 +294,65 @@ def _get_active_pairs(graph: nx.Graph, grid_layer: GridInteractionLayer):
 
 @dataclasses.dataclass(frozen=True)
 class CircuitLibraryCombination:
-    """For a given layer (which defines a set of pairs of qubits), `combinations` is a 2d array
+    """For a given layer (specifically, a set of pairs of qubits), `combinations` is a 2d array
     of shape (n_combinations, len(pairs)) where each row represents a combination (with replacement)
     of two-qubit circuits. The actual values are indices into a list of library circuits.
 
-    `layer` is used for record-keeping but is otherwise unnecessary. Therefore, it is optional.
+    `layer` is used for record-keeping. This is the GridInteractionLayer if using
+    `get_random_combinations_for_device`, the Moment if using
+    `get_random_combinations_for_layer_circuit` and ommitted if using
+    `get_random_combinations_for_pairs`.
     """
 
-    layer: Optional[GridInteractionLayer]
+    layer: Optional[Any]
     combinations: np.array
-    pairs: List[Tuple['cirq.Qid', 'cirq.Qid']]
+    pairs: List[QidPairT]
+
+
+def _get_random_combinations(
+    n_library_circuits: int,
+    n_combinations: int,
+    *,
+    pair_gen: Iterator[Tuple[List[QidPairT], Any]],
+    random_state: 'cirq.RANDOM_STATE_OR_SEED_LIKE' = None,
+) -> List[CircuitLibraryCombination]:
+    """For qubit pairs, prepare a set of combinations to efficiently sample
+    parallel two-qubit XEB circuits.
+
+    This helper function should be called by one of `xx`, `xx` which define
```

Comment:
What does "one of `xx`" mean?  Should you just remove that part and make it:  "This helper function is called by methods that define the appropriate pair_gen arguments"

Reply:
Oh man good spot. I wrote this part of the docstring before I had finalized the wrapper methods and probably meant to go back and fix it :doh: